archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ast/Input.java

```diff
@@ -692,6 +790,101 @@ public String toString() {
         }
     }
 
+    /**
+     * List of Validation.
+     */
+    public static final class Validations extends Input {
```

Comment:
This is mirroring `Transformation extends Output` ; `Output` only groups blocks, it does not have any real abstraction.
`Input` groups blocks but also has common attributes for inputs `name`, `description`, `help`, `prompt`.

Hence this should just be a block, not an input.